README.md

```diff
@@ -62,7 +62,7 @@ And add this line to your welcome view:
 
      <%= link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path %>
 
-If you visit your page and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
+If you visit your page(`users/sign_in`) and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
```

Comment:
@mendelB I believe "your page" here is not referring to the `users/sign_in` page but instead to the welcome view, referenced two lines above